### PR TITLE
Present instances in a table with sorting & filtering

### PR DESCRIPTION
**Pull request by jpnurmi**
Note: This comes without a name search/filter. It will be implemented as the next step.

[table-view.webm](https://user-images.githubusercontent.com/140617/204569797-56e772ba-af68-4cae-b24b-c68d117c4e40.webm)

Ref: #186

### CODE REVIEW

File: lib/home/instance_table_model.dart [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          return compareName(a, b);
+        case InstanceTableColumn.project:
+          return compareProject(a, b);
+      }
+    });
+    notifyListeners();
+  }
+
+  Future<void> init() async {
+    _projects = await _service.getProjects();
```
Comment (jpnurmi) [inline lib/home/instance_table_model.dart:80]: goes out of date

File: lib/home/instance_table_view.dart [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+            padding: EdgeInsetsDirectional.only(end: 4),
+            child: Text('Instance'),
+          ),
+          onSort: _onSort,
+        ),
+        if (model.selectedProjects.length > 1)
+          DataColumn2(
+            label: const Padding(
+              padding: EdgeInsetsDirectional.only(end: 4),
+              child: Text('Project'),
```
Comment (jpnurmi) [inline lib/home/instance_table_view.dart:74]: l10n

File: lib/home/instance_table_view.dart [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+      sortAscending: model.sortAscending,
+      sortColumnIndex: model.sortColumn.index,
+      columnSpacing: 10,
+      horizontalMargin: 10,
+      headingRowHeight: 46,
+      columns: [
+        DataColumn2(
+          label: const Padding(
+            padding: EdgeInsetsDirectional.only(end: 4),
+            child: Text('Instance'),
```
Comment (jpnurmi) [inline lib/home/instance_table_view.dart:66]: l10n

**Comment by d-loose** [comment]
Looks great!
But there seems to be a bug with the columns:

https://user-images.githubusercontent.com/113362648/205011500-6d9d9a06-ef31-4cbb-baa9-4bce5583f9e7.mp4

**Review by d-loose — APPROVED** [review]
great, LGTM :+1:

**Pull request merged**